.github/workflows/pr_check_load_test.yml

```diff
@@ -53,7 +53,9 @@ jobs:
       run: |
         npm install --include=dev -f
         npm ci
-
+        npm install typescript -g 
```

Comment:
Can we add typescript to package.json instead of installing it separately?

Reply:
we have the typescript in the package but somehow its not working with npm ci and npm save dev too, lets see it later